add test metrics in ci capability (#5774)
fishtown-analytics/docs.getdbt.com: website/docs/guides/set-up-ci.md

To be able to find modified nodes, dbt needs to have something to compare against. dbt Cloud uses the last successful run of any job in your Production environment as its [comparison state](/reference/node-selection/syntax#about-node-selection). As long as you identified your Production environment in Step 2, you won't need to touch this. If you didn't, pick the right environment from the dropdown.

:::info Use CI to test your metrics

> If you've [built metric definition models](/docs/build/build-metrics-intro) in your dbt project, you can [test and validate them in CI](/docs/deploy/ci-jobs#test-metrics-in-ci) to ensure code changes made to dbt models don't break these metrics.

Author: ```suggestion
If you've [built semantic nodes](/docs/build/build-metrics-intro) in your dbt project, you can [validate them in a CI job](/docs/deploy/ci-jobs#semantic-validations-in-ci) to ensure code changes made to dbt models don't break these metrics.
```
